Drivers/CAN/can.c

```diff
@@ -0,0 +1,289 @@
+/*
+    CAN DRIVER Source File
+    Link to part Datasheet for reference:
+    https://www.st.com/resource/en/user_manual/um1725-description-of-stm32f4-hal-and-lowlayer-drivers-stmicroelectronics.pdf
+    
+    Author: Hamza Iqbal
+*/
+
+#include "can.h"
+#include "can_config.h"
+
+void CAN_Msp_Init(CAN_HandleTypeDef* can_h, uint8_t line)
```

Comment:
I'd like you to go thru and rename stuff based on the conventions I set out in the IMU driver. I can draft up a confluence doc if you'd like.